cpp/src/ice2slice/Main.cpp

```diff
@@ -113,6 +113,10 @@ compile(const vector<string>& argv)
     bool preprocess = opts.isSet("E");
 
     string outputDir = opts.optArg("output-dir");
+    if (outputDir.empty())
```

Comment:
This was broken, not emitting any files when you don't specify the output dir. But the compiler din't give any errors, which seems odd.

Reply:
Huh, that's a little weird. It just passes this into `libSlice`, so there could be something weird going on downstream.
I'll take a peek at it later.